docs/integrations/sources/linkedin-ads.md

```diff
@@ -1,60 +1,61 @@
+1. [Log in to LinkedIn](https://developer.linkedin.com/) with a developer account.
+2. Click the **Create App** icon on the center of the page or [use here](https://www.linkedin.com/developers/apps). Fill in the required fields:
+   <ol type=" I">
+    <li>For <b>App Name</b>, enter a name.</li>
+    <li>For <b>LinkedIn Page</b>, enter your company's name or LinkedIn Company Page URL.</li>
+    <li>For <b>Privacy policy URL</b>, enter the link to your company's privacy policy.</li>
+    <li>For <b>App logo</b>, upload your company's logo.</li>
+    <li>For <b>Legal Agreement</b>, select **I have read and agree to these terms**.</li>
+    <li>Click <b>Create App</b>, on the bottom right of the screen. LinkedIn redirects you to a page showing the details of your application.</li>
+  </ol>
+
+3. Verify your app. You can verify your app using the following steps:
+   <ol type="I">
+    <li>
+      To display the settings page, click the <b>Settings</b> tab. On the <b>App Settings</b> section, click <b>Verify</b> under <b>Company</b>. A popup window will be displayed. To generate the verification URL, click on <b>Generate URL</b>, then copy and send the URL to the Page Admin (this may be you). Click on <b>I'm done</b>.
+      
+      If you are the administrator of your Page, simply run the URL in a new tab (if not, an administrator will have to do the next step). Click on <b>Verify</b>. Finally, Refresh the tab of app creation, the app should now be associated with your Page.
+    </li>
+    <li>
+      To display the Products page, click the <b>Product</b> tab. For <b>Marketing Developer Platform</b> click on the <b>Request access</b>. A popup window will be displayed. Review and Select <b>I have read and agree to these terms</b>. Finally, click <b>Request access</b>. 
+    </li>
+    <li>To authorize your application, click the <b>Auth</b> tab. The authentication page is displayed. Copy the <b>client_id</b> and <b>client_secret</b> (for later steps). For <b>Oauth 2.0 settings</b>, Provide a <b>redirect_uri</b> (for later steps)</li>
+    <li>
+      Click and review the <b>Analytics</b> tab. This page shows the daily application and user/member limits with the percent used for each resource endpoint.
+    </li>
+   </ol>
+
+
+4. (Optional for Airbyte Cloud) Authorize your app. In case your authorization expires:
+
+   The authorization token `lasts 60-days before expiring`. The connector app will need to be reauthorized when the authorization token expires.
+   Create an Authorization URL with the following steps:
+
+   I. Replace the highlighted parameters `YOUR_CLIENT_ID` and `YOUR_REDIRECT_URI` in the URL (`https://www.linkedin.com/oauth/v2/authorization?response_type=code&client_id=YOUR_CLIENT_ID&redirect_uri=YOUR_REDIRECT_URI&scope=r_emailaddress,r_liteprofile,r_ads,r_ads_reporting,r_organization_social`) from the scope obtain below.
```

Comment:
Why are we using roman numerals (I, II, III)?

Reply:
I used it to indicate the steps. Or should I use the ordered list tag (<ol>)? What do you think?

Reply:
Just use the regular (1, 2, 3..) numbered list

Reply:
Resolved. Thanks